tests/gs-self-test.c

```diff
@@ -182,7 +184,6 @@ gs_plugins_apk_app_install_remove (GsPluginLoader *plugin_loader)
 
   // make sure we got the correct app and is managed by us
   g_assert_cmpstr (gs_app_get_id (app), ==, "apk-test-app.desktop");
-  g_assert_cmpstr (gs_app_get_management_plugin (app), ==, "apk");
```

Comment:
Here we should also still check that the management plugin is the correct one :)